SimG4Core/Application/python/g4SimHits_cfi.py

```diff
@@ -575,6 +575,23 @@
 run2_common.toModify( g4SimHits.HFShowerLibrary, FileName = 'SimG4CMS/Calo/data/HFShowerLibrary_npmt_noatt_eta4_16en_v4.root' )
 run2_common.toModify( g4SimHits.HFShower, ProbMax = 0.5)
 
+
+from Configuration.Eras.Modifier_ctpps_2021_cff import ctpps_2021
```

Comment:
@mundim , if the modifier is configured in this way, then central detector simulation is disabled and only PPS simulation is enabled. In that case, this modifier should be applied only for specific ctpps_2021 simulation. I assume it is the case and in the normal WF used in tests this should not be used.

Reply:
Hi @civanch (sorry for the late reply, my laptop had to be taken to the maintenance, I'm using a tablet).

I'm not sure I understood the problem. I just picked up from the comments in the PR 29916, as suggested by @makortel for Run2  (https://github.com/cms-sw/cmssw/pull/29916#discussion_r428796936) and replaced ctpps_2016 by ctpps_2021.

Do I have to make others changes or the rebase as your next message is enough?

Reply:
@civanch Could you clarify in which way the customization below disables central detector simulation? Is it perhaps the `g4simhits.Generator.HepMCProductLabel = 'LHCTransport`?

Reply:
I don' t believe so. We have done private pps simulation with this label and the central CMS simulation was not disabled.